kubernetes-manifests/loadgenerator.yaml

```diff
@@ -29,6 +29,12 @@ spec:
     spec:
       terminationGracePeriodSeconds: 5
       restartPolicy: Always
+      initContainers:
+      - name: init-frontend-ready
+        image: busybox:1.28
+        command: ['/bin/sh', '-c']
+        args:
+          - until  [[ "$(wget --spider -S http://frontend 2>&1 | grep HTTP/)" == "  HTTP/1.1 200 OK" ]]; do echo waiting for frontend; sleep 2; done
```

Comment:
We actually had this. But we had to remove it because of istio.

Reply:
See #279.
https://github.com/GoogleCloudPlatform/microservices-demo/commit/8cfb88bf2f4eb1843b08b45138b4020b5681493f#diff-cbd8d0a4b56cb8944c9951eac8e9104f

Let's not add it again.

Reply:
@askmeegs Users are interpreting the load generator crash loop as a bug. Is there anything we could do to make the InitContainer play nice with mTLS? Or is it not worth the trouble?